### PR TITLE
Feature/font

### PR DESCRIPTION
**Pull request by ZenVega**
replace CityLAB font with TSG standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +70,13 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
 
 # tsb-jahresbericht-2021
+
+## Fonts
+
+The fonts used in this project are not public. As a member of Technologiestiftung you can access them via
+
+Google Drive: \_CityLAB Berlin/Visuals/06 Design/tsg-standard-font Clan\
+or\
+Server Space: LWS$/Öfentlichkeitsarbeit/Jahresberichte/21_Jahresbericht\
```
Comment (MRigal) [inline README.md]: typo `Öffentlichkeitsarbeit` and they are actually currently not there :-D

Also I think they are somewhere else already on that drive but I don't know where :-D 
@marcoka2000 maybe you know?

**Review by MRigal — CHANGES_REQUESTED** [review]
mini-typo and let's wait if Marco can find the original place of the fonts on LWS

File: README.md (hunk 1 above)
Comment (ZenVega) [reply README.md]: ah true @marcoka2000 told me where they are i mixed it up. Will take care of it.

**Review by MRigal — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**